### PR TITLE
Fix app labels

### PR DESCRIPTION
**Pull request by InfoSec812**
Add labels metadata to templates so that applications are displayed properly in OCP console

### CODE REVIEW

**Comment by mdanter** [comment]
@InfoSec812 Can we standardize on json for templates?

**Comment by InfoSec812** [comment]
@mdanter Eric and Øystein are recommending YAML as the default. That said, I really have no preference. The problem is that once I extract and customize the template it is difficult to convert from one to the other. Suggestions for a way to easily convert everything to one or the other format would be greatly appreciated. The tools I have tried fail to be effective in either direction.

File: templates/jenkins-ephemeral/template.yml [hunk 1]
```diff
@@ -0,0 +1,211 @@
+apiVersion: v1
```
Comment (sherl0cks) [inline templates/jenkins-ephemeral/template.yml]: We should be using the out of the box template, and if it is insufficient, opening bugs to that end. I have opened https://bugzilla.redhat.com/show_bug.cgi?id=1503260

Please remove this template.

**Comment by sherl0cks** [comment]
@InfoSec812 as to json vs yaml, I don't really care either. I'm fine with JSON going forward, but in this case, I don't think its a problem once my requested changes are made

**Comment by sherl0cks** [comment]
Per https://bugzilla.redhat.com/show_bug.cgi?id=1503260, we need to update our automation to apply this label. I've opened https://github.com/redhat-cop/casl-ansible/issues/116 accordingly. Closing this PR

**Pull request closed without merging**